torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -12,6 +12,18 @@ PythonRpcHandler::PythonRpcHandler() {
   loadResultFunction_ = module.attr("load_python_udf_result_internal");
 }
 
+void PythonRpcHandler::cleanUp() {
+  AutoGIL ag;
+  if (!runUDFFunction_.is_none()) {
+    runUDFFunction_.dec_ref();
```

Comment:
Could you please remind me where is the corresponding `inc_ref` for this? Is it when we load the function in `module.attr`? I am a little confused of why we need to do `dec_ref` on our own but not `inc_ref`?